executor/compiler.go

```diff
@@ -367,3 +384,138 @@ func GetInfoSchema(ctx sessionctx.Context) infoschema.InfoSchema {
 	}
 	return is
 }
+
+func addHint(ctx sessionctx.Context, stmtNode ast.StmtNode) ast.StmtNode {
+	switch x := stmtNode.(type) {
+	case *ast.ExplainStmt:
+		switch x.Stmt.(type) {
+		case *ast.SelectStmt:
+			normalizeExplainSQL := parser.Normalize(x.Text())
```

Comment:
We need to add a session variable to control the use of bind, or it will affect OLTP performances a lot.